command/app.go

```diff
@@ -143,7 +143,11 @@ var app = &cli.App{
 	},
 	After: func(c *cli.Context) error {
 		if c.Bool("stat") {
```

Comment:
nit: this might be a single line. also no need to add issue reference in comment

```suggestion
		if c.Bool("stat") && len(stat.Statistics()) > 0 {
```

Reply:
Fixed 👍